misc/gce-federation/inventory/group_vars/federation

```diff
@@ -0,0 +1,27 @@
+---
+# These two really should have some sane defaults rather than failing if not specified
+ansible_become: yes
+ansible_ssh_user: jdetiber
```

Comment:
@detiber I think the only NITs I have would be if we can make the ssh user generic. Everything else looks really good